wapitiCore/attack/mod_exec.py

```diff
@@ -168,7 +168,7 @@ async def attack(self, request: Request, response: Optional[Response] = None):
                 vuln_info = None
 
                 # No timeout raised, check for patterns in response
-                if any(rule in response.content for rule in payload_info.rules):
+                if any(rule.replace("<SPACE>", " ") in response.content for rule in payload_info.rules):
```

Comment:
You should use `[SPACE]` instead, as other ini files are using square brackets, otherwise it looks good

Reply:
thanks @devl00p, just updated my code with `[SPACE]` instead of `<SPACE>`